packages/electron-redux/src/middleware/__tests__/forwardToMain.js

```diff
@@ -73,3 +73,44 @@ describe('forwardToMain', () => {
     expect(next).toHaveBeenCalledTimes(0);
   });
 });
+
+describe('forwardToMainWithParams', () => {
+  beforeEach(() => {
+    jest.clearAllMocks();
```

Comment:
this is probably not necessary

Reply:
I need to reset `ipcRenderer.send` call counter.